sdk/include/reactos/mc/neteventmsg.mc

```diff
@@ -119,6 +132,9 @@ Un paquet de pilote reçu par le sous-système IO était invalide. La donnée es
 Language=Polish
 Pakiet sterownika otrzymany z podsystemu wejścia/wyjścia jest niepoprawny. Przedstawione dane to otrzymany pakiet.
 .
+Language=Romanian
+Un pachet primit al modulului pilot din subsistemul de intrare/ieșire a fost nevalid. Datele sunt pachetul însuși.
```

Comment:
The above comment is not a call for change.

Reply:
Shouldn't it be "In/Ie", instead of "In/Ex"? It seems to be more explicit."Ex" reffers to "Exit".

Reply:
I remember that somewhere I translated with "In/Ie".

Reply:
My reason, like I mentioned above, was about the direction of the communication. "In/Ex" are short for "Interior/Exterior", and there are quite a few resources translated this way (like [machine.inf](https://github.com/reactos/reactos/blob/master/media/inf/machine.inf#L613), [cmd](https://github.com/reactos/reactos/blob/master/base/shell/cmd/lang/ro-RO.rc#L107), [dxdiag](https://github.com/reactos/reactos/blob/master/base/applications/dxdiag/lang/ro-RO.rc#L255), [taskmgr](https://github.com/reactos/reactos/blob/master/base/applications/taskmgr/lang/ro-RO.rc#L311), [devmgr](https://github.com/reactos/reactos/blob/master/dll/win32/devmgr/lang/ro-RO.rc#L42), [errcodes.mc](https://github.com/reactos/reactos/blob/master/sdk/include/reactos/mc/errcodes.mc#L1708), and probably more). Since you took on maintenance of ReactOS Romanian resources, now it's up to you to decide if it makes sense to you this way or you'd rather want to improve (be it with "In/Ie" or with something else). The important thing is (as you'd already guess) to keep things consistent across the body of translation work.

Reply:
Yes, you're right. I completely forgot about it.